src/topology/algebra/order/lower_topology.lean

```diff
@@ -0,0 +1,144 @@
+/-
+Copyright (c) 2022 Christopher Hoskin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Christopher Hoskin
+-/
+import order.upper_lower
+import topology.separation
+
+/-!
+# Lower topology
+
+This file introduces the lower topology on a preorder. It is shown that the lower topology on a
+partial order is T₀ and the complements of the upper closures of finite subsets form a basis.
+
+## Implementation notes
+
+Approach inspired by `order_topology`.
```

Comment:
Given that you are quite exactly *not* following the design of `order_topology`, I would replace this message by an explanation of the difference between the two approaches.